### DISCUSSION TITLE
Pulsed RGB capability

### DISCUSSION THREAD

**Discussion by Klankowski** [forkineye/ESPixelStick#387] [Ideas]
This may be a very novel concept, but these two wire Pulsed RGB's are common in novelty lights and now ropelight and 5mm strings, i am looking to adapt the ESP module to output a signla for a pulse modulation of the power delivery operate the sting of light.  These lights are similar to minleon's Twinkly Pro series of strings.  Could you help me evolve this idea some more?

https://youtu.be/WLXnCaXX7w4
https://youtu.be/gxdgRHmOuIo
https://youtu.be/iuZZxOvY9Ls

**Comment by MartinMueller2003** [comment]
If you can identify the protocol used, then we might be able to make something happen.
